mevislab.github.io/content/tutorials/thirdparty/matplotlib/modulesetup.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Example 1: Setting up the module"
+date: 2023-05-26
+status: "OK"
+draft: false
+weight: 881
+tags: ["Advanced", "Tutorial", "Matplotlib", "Visualization"]
+menu: 
+  main:
+    identifier: "matplotlibexample1"
+    title: "Example 1: Setting up the module"
+    weight: 881
+    parent: "matplotlib"
+---
+
+# Example 1: Setting up the module
+
+To be able to acess the data needed for our grayscale distribution plots, we need a network consisting of a module that imports a DICOM, a module that differentiates between slices and another that ouputs histogram data. 
+
+So open up your MeVisLab workspace and add the modules `LocalImage`, `SubImage` and `Histogram` to it.
+Connect the output of `LocalImage` to the input of `SubImage` and the output of `SubImage` with the input of `Histogram`.
+If you feel like using a shortcut, you can also download the base network below and open it in your MeVisLab.
+
+Your finished network should look like this: 
+
+![Base network](/images/tutorials/thirdparty/Matplotlib1.PNG)
+
+{{< networkfile "/tutorials/thirdparty/matplotlib/MatplotlibBaseNetwork.mlab" >}}
+
+### Excursion on the concept behind modules
```

Comment:
Are you sure to explain that here again? Wouldnt it be sufficient to reference the existing tutorials?

Reply:
Added it because it's directly related to everything that follows - 
But I could also remove it if you think it's too long or not necessary.